console-ui/src/design/portal.css

```
@@ -2484,40 +2658,214 @@ body {
+   CJK reads denser than Latin — use sans + 1.85 line-height + generous
+   block spacing rather than the page-level .reading serif. */
 .answer-text {
-  font-size: var(--ovp-text-sm);
-  line-height: 1.65;
+  font-family: var(--font-sans, var(--ovp-font-sans, inherit));
+  font-size: 1.05rem;
+  line-height: 1.85;
+  color: var(--text, inherit);
   overflow-wrap: anywhere;
+  letter-spacing: 0.015em;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)